php/auth.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$ERRORLOG = getenv('PHP_ERROR_LOG');
```

Comment:
Oh ya. Getting mixed up with the python rewrite (since on my dev machine I removed lighttpd, php can only write to /var/log/nginx/).